provider/testdata/Cargo.toml

```diff
@@ -137,7 +137,134 @@ cldr_json_glob = [
 ]
 
 # Git tag or sha1 for the CLDR data used to generate the testdata.
-gitref = "41.0.0"
+cldr_json_gitref = "41.0.0"
+
+icuexportdata_glob = [
```

Comment:
Right now this is just an exhaustive list of everything that was currently being included. We can globify it later if needed.